templates/docs/patterns/navigation/index.md

```diff
@@ -86,6 +86,21 @@ When used on websites with more complex information archutecture there may be a
 
 <div class="embedded-example"><a href="/docs/examples/patterns/navigation/reduced" class="js-example">View example of the reduced navigation pattern</a></div>
 
+## Full-width dropdowns
+
+<div class="p-notification--caution">
+  <div class="p-notification__content">
+    <h3 class="p-notification__title">Work in progress:</h3>
+    <p class="p-notification__message">Full-width dropdowns are part of a larger piece of work to upstream the ubuntu.com navigation to Vanilla. These changes are introduced gradualy and may not be fully functional or documented yet.</p>
+  </div>
+</div>
+
+Instead of standard navigation dropdown menus a full-width dropdowns can be used to provide more space for the content of the dropdown.
+
+To use them add `.is-full-width` class name to `.p-navigation__dropdown` element. Inside of it you need two containers for full-width desktop dropdown content `.p-navigation__dropdown-content--full-width` and mobile sliding navigation `.p-navigation__dropdown-content--sliding` content. The mobile content will be displayed on small screens only and desktop content on larger screens (based on the value of `$breakpoint-navigation-threshold`).
+
+<div class="embedded-example"><a href="/docs/examples/patterns/navigation/dropdown-full-width" class="js-example">View example of the full-width dropdown pattern</a></div>
```

Comment:
There seems to be some whitespace control issues in the rendered markup here:

![image](https://github.com/user-attachments/assets/adb05cd0-6e72-42e7-bbaf-e1c01b22780e)

Though I think this is OK considering this is marked as work-in-progress and there is other work dependent on this; I'd rather not hold the PR up over this

Reply:
Parts of this are generated from macro, so I guess macro defines the indentation, not the parent HTML.

I suppose the idea to have external formatter would be good in long term, not to rely on how HTML source is structured.

Reply:
cc @pastelcyborg related to our previous conversation about external formatter